rust/src/cliwrap/kernel_install.rs

```diff
@@ -64,6 +64,20 @@ fn redo_systemctl_wrap() -> Result<()> {
     Ok(())
 }
 
+#[context("Running initoverlayfs")]
+fn run_initoverlayfs() -> Result<()> {
+    const INITOVERLAY_INSTALL_CMD: &str = "initoverlayfs-install";
+
+    let initoverlayfs_install = Path::new("/usr/bin/initoverlayfs-install");
+    if initoverlayfs_install.try_exists()? {
```

Comment:
So now that initoverlayfs-install is a direct swappable replacement for dracut, we can just set the dracut_path variable as /usr/bin/initoverlayfs-install, in the initoverlayfs case...